extensions/gdx-setup/res/com/badlogic/gdx/setup/resources/android/AndroidManifest.xml

```diff
@@ -1,18 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="%PACKAGE%" >
+    xmlns:tools="http://schemas.android.com/tools"
+    package="%PACKAGE%">
+
+    <uses-feature android:glEsVersion="0x00020000" android:required="true" />
 
     <application
         android:allowBackup="true"
+        android:fullBackupContent="true"
         android:icon="@drawable/ic_launcher"
         android:isGame="true"
         android:appCategory="game"
-        android:label="@string/app_name">
+        android:label="@string/app_name"
+        tools:ignore="UnusedAttribute">
```

Comment:
Alternatively `tools:targetApi` can be used. Either way, this stops it complaining about about `isGame` and `appCategory` only being used on newer Android versions.